content/shmem_team_split_strided.tex

```diff
@@ -13,8 +13,7 @@
 \begin{apiarguments}
 \apiargument{IN}{parent\_team}{An \openshmem team.}
 
-\apiargument{IN}{start}{The lowest \ac{PE} number of the subset of \acp{PE} from
-the parent team that will form the new team.}
+\apiargument{IN}{start}{The first \acs{PE} number of the subset of \acp{PE} from the parent team that will form the new team. If the stride is less than zero, the first \acs{PE} number is the highest; if greater than zero, it is the lowest; if the stride is zero, it is the starting \acs{PE}.}
```

Comment:
I'm okay with this language if others think its clear enough. My concern was that we never say what highest/lowest is in regard to. I know this is clarified later.

My suggestion is to add clarity to highest: `the first \acs{PE} number is the highest \acs{PE} of the parent team`

Reply:
How about https://github.com/wokuno/specification/pull/1/commits/1795f124c8e4ca3bbd34a3cf464840e6624be36f?